web-api/src/main/java/com/graphhopper/jackson/LMProfileConfigMixIn.java

```diff
@@ -21,6 +21,10 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 public interface LMProfileConfigMixIn {
+    // todonow: maybe both of these are no longer needed because of snake case module?
```

Comment:
The first one is likely still required as snake might do maximum_l_m_weight ... but not tested this.

Reply:
Exactly, without the rule here Jackson apparently expects `maximum_lmweight`, so the second uppercase letter does not introduce a second underscore, but also the third uppercase letter (W in Weight) does not add an underscore.

Reply:
Aha, thanks!